app/js/controllers/foyer/ressources/rfr.js

```diff
@@ -3,7 +3,7 @@
 angular.module('ddsApp').controller('FoyerRessourceRfrCtrl', function($scope) {
     $scope.yearMoins2 = moment($scope.situation.dateDeValeur).subtract('years', 2).format('YYYY');
     $scope.yearMoins1 = moment($scope.situation.dateDeValeur).subtract('years', 1).format('YYYY');
-    $scope.rfr = $scope.situation.rfr || 0;
+    $scope.rfr = $scope.situation.rfr || undefined;
```

Comment:
I don't think that's what you want to do. Setting a default value to `undefined` doesn't have much sense except preventing falsies.

In the case at hand: if `$scope.situation.rfr` is already `undefined`, this is useless. However, if the user explicitly entered `0`, you're ignoring the input (`0` is falsy, thus the value is the next operand).